contracts/MyToken.sol

```diff
@@ -1,17 +1,35 @@
 pragma solidity ^0.4.24;
 
 import "openzeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "./DSToken.sol";
 
 contract MyToken is ERC721Token {
 
+  address daiAddress;
+
   constructor (string _name, string _symbol) public ERC721Token(_name, _symbol) {
+  }
+
 
+  function setDaiAddress(address _addr) public {
+    daiAddress = _addr;
   }
 
   function buyBook() public payable {
     require(msg.value == 1 ether);
     uint256 newTokenId = super.totalSupply() + 1;
     super._mint(msg.sender, newTokenId);
   }
+
+  function getDaiAddress() public returns (address) {
```

Comment:
I don't see this used in the test. I assume this is just for making sure you have the right DAI address?

Reply:
yah, might be good to have in case someone needs to check. Using it would be redundant in the tests because ```setDaiAddress``` already ensures it's the same address.